app/assets/stylesheets/style.scss

```diff
@@ -867,4 +873,21 @@ label:hover {
   }
 }
 
+.notice_wrapper {
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: 2000;
+  width: 100%;
+  background: rgba(137, 59, 100, 0.7);
+  .notice, .alert {
+    padding: 2rem 0;
```

Comment:
Properties should be ordered color, font-size, font-weight, letter-spacing, margin, padding, text-align